### PR TITLE
add visual effect when dragging the tree

### LINKED ISSUE #69: Adding visual effect for tree map box
adding a resize feature in the tree map box will be a good idea I think, cause while scrolling through the browser in smaller screens like in phone, tree map box acquires almost the whole screen leaving some spaces below and above it, and only those spaces can be used for scrolling which makes it difficult for the user to scroll down or up.
IMG_20210401_173607
Only red area marked can be used for scrolling down, which seems frustrating, I felt this when I used Edam browser in the phone for the first time, 
I thought my phone isn't working :( took me few minutes to realise that box isn't scrollable like other elements. Even scrolling using vertical scrollbar isn't at all feasible in phone at least because of its less width by default.
 

So a decrease option to minimise the box size( not to zero)  will be nice.

1) It can be done something similar to what's there for applications in an os, Resizing using cursor at the corners. 
images

2) Another way can be to provide a button kind of thing like in some PDF reader to change size to 50%, 100%, 10%... For the tree map box only. 
IMG_20210401_174557

I think 1 will work in desktop version only, so option 2 can be used which will be made to appear for screens of size less than a particular value. 

Refreshing is also difficult currently.

What are your views @bryan-brancotte @matuskalas @hmenager :-)

### LINKED ISSUE #105: Propose changing the mouse cursor to a hand when moving the tree around
@matuskalas @hmenager @bryan-brancotte It could be useful to have a hand cursor icon when interacting with the tree instead of the mouse cursor to alert the user that they're able to pull and push the tree around. When hovering over the nodes on the tree, a pointer cursor appears, but outside of that there is only the mouse cursor. 

Would having a hand/grabbing cursor be more helpful instead?

### PR DESCRIPTION
**Pull request by bryan-brancotte**
### Checklist

- [x] I indicated which issue (if any) is closed with this PR using [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] I only changed lines related to my PR (no bulk reformatting)
- [x] I indicated the source and check the license if I re-use code, or I did not re-use code
- [x] I made my best to solve only one issue in this PR, or explain why multi had to be solved at once.

### Issue

resolve #69   
closes #105

### Details
works in Firefox and Chrome

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -214,6 +214,12 @@ body {
 .temp:last-child {
     opacity: 0;
 }
+svg{
+    transition: box-shadow 0.1s ease-in-out;
+}
+.dragged-or-moved{
+    box-shadow: inset 0 0 0.5em 0.5em #6666;
+}
```
Comment (HagerDakroury) [inline css/index.css:222]: would it be better to decrease the frame size? like 0.2em. I believe it'll be less distracting.

**Review by HagerDakroury — COMMENTED** [review]
I can't really test that using chrome's dev tools. Do you have any advice on what to do to test?

File: css/index.css (hunk 1 above)
Comment (bryan-brancotte) [reply css/index.css:222]: Peek 26-05-2021 12-32
 
I haven't tested it on an actual smarphone, but confident that there will be no side effect as we use the d3.js event that are already used to drag and zoom

Comment (HagerDakroury) [reply css/index.css:222]: makes sense that it would still work yes, Awesome, this looks good to merge!

**Comment by HagerDakroury** [comment]
tested on my phone and it's working. Nice! 🎉

**Comment by bryan-brancotte** [comment]
working well on smartphone!

@HagerDakroury as you pointed out that it could be distracting, what do you think of not having the effect when a mouse is connected ? Similarly to #205

**Comment by HagerDakroury** [comment]
I'm not that bothered by the effect but I don't know how it might feel for a frequent user. Might need @matuskalas opting in here.
 
If it's removed for mouses though, we should add the drag cursor #105 alternatively (or along with)

**Comment by bryan-brancotte** [comment]
If for the both of us it's ok we can keep as is. I we get feedback on it we will open an issue to improve it

**Pull request merged**